lang/java/reef-bridge-java/src/main/java/org/apache/reef/javabridge/generic/JobDriver.java

```diff
@@ -590,7 +607,8 @@ public void onNext(final StartTime startTime) {
           LOG.log(Level.INFO, "Finished CLR bridge setup for {0}", startTime);
         }
 
-        NativeInterop.callClrSystemOnStartHandler();
+        bridge.callClrSystemOnStartHandler();
```

Comment:
In .Net, you have a new CLR Bridge project for the transition. In Java it is replaced with the new approach directly. Shall we keep both old and new way so that we can do some perf comparison?